src/components/AccountLoginForm.jsx

```diff
@@ -25,38 +27,51 @@ const AccountLoginForm = ({ t, isOAuth, oAuthTerminated, fields, error, dirty, s
           const description = fields[name].hasDescription
             ? <ReactMarkdownWrapper source={t(`connector.${connectorSlug}.description.field.${name}`)} />
             : ''
+          const readOnly = name === 'login' && isUpdate
+
+          let fieldReact
           switch (fields[name].type) {
             case 'password':
-              return <div>
+              fieldReact = <div>
                 {description}
                 <PasswordField
                   label={t(`account.form.label.${name}`)}
                   name={inputName}
                   placeholder={t('account.form.placeholder.password')}
                   invalid={!!error}
+                  giveFocus={!hasFocused}
```

Comment:
You could directly use `giveFocus` variable here in order to be more explicit I think.